projects/swimlane/ngx-charts/src/lib/pie-chart/pie-chart.component.scss

```diff
@@ -32,3 +32,14 @@
     }
   }
 }
+
+.pie-chart {
+  .percent-label {
+    font-size: 24px;
+    font-weight: 400;
+  }
+
+  .pieTotal {
+    font-weight: 400;
```

Comment:
Question: do we really need this class? Is it default weight?

Reply:
It's not default weight. The default weight is too light/thin:
<img width="440" alt="Screenshot 2023-12-24 at 11 32 35 AM" src="https://github.com/swimlane/ngx-charts/assets/47932418/1474f7d3-2f0e-42b8-aba5-5405656a2f6e">